test/test_managercli.py

```diff
@@ -337,6 +337,24 @@ class NoStrException(Exception):
                 pass
         self.assertEqual("%s\n\n" % msgs[0], cap.err)
 
+    def test_only_exception(self):
+        ex = ValueError("test message")
+        with Capture() as cap:
+            try:
+                system_exit(1, ex)
+            except SystemExit:
+                pass
+        self.assertEqual("%s\n" % str(ex), cap.err)
+
+    def test_only_exception_unicode(self):
+        ex = ValueError("\u2620 \u2603 \u203D")
+        with Capture() as cap:
+            try:
+                system_exit(1, ex)
+            except SystemExit:
+                pass
+        self.assertEqual("%s\n" % str(ex), cap.err)
+
```

Comment:
I would like to see here unit test for the case, when second argument of `system_exit` is list of exceptions handled by `ExceptionMapper`.